spec/system/visit_component_show_spec.rb

```diff
@@ -8,24 +8,24 @@
 
```

Comment:
same with the filename here and we should add the tests for the sp_component then...